src/Grpc.AspNetCore.Server/Internal/ClientInterceptorContext.cs

```diff
@@ -0,0 +1,63 @@
+﻿#region Copyright notice and license
+
+// Copyright 2018 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+using System;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace Grpc.Core.Interceptors
+{
+    /// <summary>
+    /// Carries along the context associated with intercepted invocations on the client side.
+    /// </summary>
+    public struct ClientInterceptorContext<TRequest, TResponse>
```

Comment:
Double check: This is temporary and will come from Grpc.Core.Api?

Reply:
Yes https://github.com/grpc/grpc/pull/18914